src/tools/bundle-cli.ts

```diff
@@ -40,4 +40,7 @@ Examples
   process.exit();
 }
 
-void bundle(argv._, Array.isArray(argv.o) ? argv.o[0] : argv.o, argv.verbose);
+void bundle(argv._, Array.isArray(argv.o) ? argv.o[0] : argv.o, argv.verbose).catch(e => {
```

Comment:
Its not clear to me how this works. What is argv.o?

Reply:
I've changed it a bit, hopefully for better. `argv` holds arguments. argv.o === argv.output (-o vs --output) and specifies the path to the resulting bundle. PTAL.

Reply:
I think we can put this down to my lack of experience with this api, I won't block on it anyway.